gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/tasks/CreateManifestTask.java

```diff
@@ -342,7 +342,13 @@ private void ensureLockfileIsUpToDate(List<ProductDependency> productDeps) {
                                 recommendedDep.getProductName(),
                                 recommendedDep.getMinimumVersion(),
                                 recommendedDep.getMaximumVersion(),
-                                recommendedDep.getRecommendedVersion()));
+                                recommendedDep.getRecommendedVersion()))
+                        .peek(productDependency -> log.debug(
```

Comment:
debug is actually crazily noisy - shall we take this up to info?

```suggestion
                        .peek(productDependency -> log.info(
```